docs/gettingstarted/mk8s/k8s-microk8s-cluster.md

```diff
@@ -0,0 +1,149 @@
+```
+
+5. Check that SC4SNMP has been scaled:
+
+```bash
+microk8s kubectl get pods -n sc4snmp
+```
+
+You should get 4 replicas for each worker and traps service:
+
+```bash
+NAME                                                          READY   STATUS      RESTARTS   AGE
+snmp-mibserver-5df74fb678-zkj9m                               1/1     Running     0          25h
+snmp-mongodb-6dc5c4f74d-xg6p7                                 2/2     Running     0          25h
+snmp-redis-master-0                                           1/1     Running     0          25h
+snmp-splunk-connect-for-snmp-inventory-k9t87                  0/1     Completed   0          3m
+snmp-splunk-connect-for-snmp-scheduler-76848cf748-57qbx       1/1     Running     0          25h
+snmp-splunk-connect-for-snmp-trap-9f55664c4-9dv7d             1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-trap-9f55664c4-crgld             1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-trap-9f55664c4-sb768             1/1     Running     0          25h
+snmp-splunk-connect-for-snmp-trap-9f55664c4-tkhcp             1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-worker-poller-7487956697-4hvpl   1/1     Running     0          21h
+snmp-splunk-connect-for-snmp-worker-poller-7487956697-8bvnn   1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-worker-poller-7487956697-9dfgt   1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-worker-poller-7487956697-hlhvz   1/1     Running     0          24h
+snmp-splunk-connect-for-snmp-worker-sender-657589666f-979d2   1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-worker-sender-657589666f-mrvg9   1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-worker-sender-657589666f-qtcr8   1/1     Running     0          21h
+snmp-splunk-connect-for-snmp-worker-sender-657589666f-tc8sv   1/1     Running     0          24h
+snmp-splunk-connect-for-snmp-worker-trap-859dc47d9b-6fbs2     1/1     Running     0          24h
+snmp-splunk-connect-for-snmp-worker-trap-859dc47d9b-kdcdb     1/1     Running     0          3m1s
+snmp-splunk-connect-for-snmp-worker-trap-859dc47d9b-sfxvb     1/1     Running     0          3m
+snmp-splunk-connect-for-snmp-worker-trap-859dc47d9b-xmmwv     1/1     Running     0          21h
+```
+
+6. For autoscaling update `values.yaml` and after that redeploy SC4SNMP:
+
+```yaml
+worker:
+  poller:
+    autoscaling:
+      enabled: true
+      minReplicas: 1
+      maxReplicas: 10
+  trap:
+    autoscaling:
+      enabled: true
+      minReplicas: 1
+      maxReplicas: 10
+  sender:
+    autoscaling:
+      enabled: true
+      minReplicas: 1
+      maxReplicas: 10
+
+traps:
+  autoscaling:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 10
+```
+
+
+## Scaling criteria
+
+if `2 * periodic_task_exec_time * inventory_size >= workers_count * task_period` you need to scale `workers_count` or increase `task_period`.
```

Comment:
writing the formula first and describing the if part later might be more readable. What do you think? Like:

`2 * periodic_task_exec_time * inventory_size >= workers_count * task_period`
where:
* `periodic_task_exec_time` - Execution time of polling / walk task.
* `inventory_size` - How many items we have on inventory.
* `workers_count` - How many workers for polling / walk we have.
* `task_period` - Walk / Polling period time.

If the left side of equation is higher you need to scale `workers_count` or increase `task_period`.